code/overview/overfitting/example.py

```diff
@@ -0,0 +1,49 @@
+print(__doc__)
```

Comment:
this code is copied directly from 
https://scikit-learn.org/stable/auto_examples/model_selection/plot_underfitting_overfitting.html